instrumentation-tests/src/androidTest/java/com/mapbox/navigation/instrumentation_tests/utils/coroutines/Adapters.kt

```diff
@@ -119,6 +122,22 @@ suspend fun MapboxNavigation.requestRoutes(options: RouteOptions) =
         }
     }
 
+suspend fun MapboxNavigation.requestAlternativeRoutesAsync() = suspendCoroutine<Expected<RouteAlternativesError, List<NavigationRoute>>> { continuation ->
+    requestAlternativeRoutes(object : NavigationRouteAlternativesRequestCallback {
+        override fun onRouteAlternativeRequestFinished(
+            routeProgress: RouteProgress,
+            alternatives: List<NavigationRoute>,
+            routerOrigin: RouterOrigin
+        ) {
+            continuation.resume(ExpectedFactory.createValue(alternatives))
+        }
+
+        override fun onRouteAlternativesRequestError(error: RouteAlternativesError) {
+            continuation.resume(ExpectedFactory.createError(error))
+        }
+    })
+}
+
```

Comment:
does somebody use it?

Reply:
No, thanks. I was just testing something.